applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_element.cpp

```diff
@@ -223,19 +223,16 @@ namespace Kratos
 
   template <unsigned int TDim>
   void TwoStepUpdatedLagrangianVPImplicitElement<TDim>::CalculateOnIntegrationPoints(const Variable<double> &rVariable,
-                                                                                    std::vector<double> &rOutput,
-                                                                                    const ProcessInfo &rCurrentProcessInfo)
+                                                                                     std::vector<double> &rOutput,
+                                                                                     const ProcessInfo &rCurrentProcessInfo)
   {
-    if (rVariable == EQ_STRAIN_RATE)
-    {
-      rOutput[0] = this->GetValue(EQ_STRAIN_RATE);
-    }
+    rOutput[0] = this->GetValue(rVariable);
```

Comment:
In order this to be **safe** you have to be sure to have `SetValue` its value before... If you are not sure you should use `this->HasValue()`